macros/models/_sector/referral/platforms/paragraph_referral_rewards.sql

```diff
@@ -35,4 +36,39 @@ where
 {% if is_incremental() %}
     and {{incremental_predicate('evt_block_time')}}
 {% endif %}
+
+UNION ALL
+
+select
+    '{{blockchain}}' as blockchain
+    ,'paragraph' as project
+    ,'v1' as version
+    ,call_block_number as block_number
+    ,call_block_time as block_time
+    ,cast(date_trunc('day',call_block_time) as date) as block_date
+    ,cast(date_trunc('month',call_block_time) as date) as block_month
+    ,call_tx_hash as tx_hash
+    ,'NFT' as category
+    ,0x0000000000000000000000000000000000000000 as referrer_address
+    ,minter as referee_address
+    ,{{native_currency_contract}} as currency_contract
+    ,0 as reward_amount_raw
+    ,t."from" as project_contract_address
+    ,cardinality(trace_address) + cast(coalesce(nullif(array_join(trace_address, ''), ''), '0') as integer) + 9999999 as sub_tx_id
```

Comment:
This is ugly but I couldn't think of a better way to try to make sure that there are no conflicts with `FeeDistributed` `evt_index` values in case there are multiple mint calls in the same transaction

Reply:
`+9999` is a fine hack to make sure there are no collisions with evt indexes.
A more straightforward way of getting an index from the trace address would be:
`row_number() over (partition by tx_hash, order by trace_address)`